keyvalue/server/pom.xml

```diff
@@ -46,7 +46,12 @@
     </dependency>
     <dependency>
       <groupId>org.alluxio</groupId>
-      <artifactId>alluxio-core-server</artifactId>
+      <artifactId>alluxio-core-server-master</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.alluxio</groupId>
+      <artifactId>alluxio-core-server-worker</artifactId>
```

Comment:
I feel like it is a difficult convention to maintain ... what are its benefits? how can we enforce it?